accounting/account/tests/factories.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# OpenCraft -- tools to aid developing and hosting free software projects
```

Comment:
Oh :( @antoviaque what should we call this?

@itsjeyd I know @antoviaque is away this week, so maybe to prevent blocking this getting merged and thus not blocking the deployment work (which'll need updated branches, etc.), let's come back to this on the next piece of work we do when we figure out an appropriate description.

Reply:
@UmanShahzad Sure, following up in a later PR (just like we're doing here with the copyright years :)) sounds good.

Reply:
Haha, that's a cool rebrand :P Crafty Instances, i.e. "CI", I see what you did there, if that was intentional.

I've already spoken about this on some ticket: I'm cool with renaming this repo with prefix `crafty-`, no preference from my side. But we still need a proper description of the project to put in these headers, which is what I think @itsjeyd's comment was originally about.

Reply:
This line includes both - the software's name (here "OpenCraft", which would also need to be changed on Ocim's side actually) and description. The later is easier imho - we could just go with something like "Accounting automation software for remote teams".

Reply:
Alrighty. So I'll wait until we finalize a name (I see the discussion going on below), then update this.

Reply:
Yup - maybe add a ticket in the backlog to not forget?

Reply:
Yep, put it on my list to do on Friday.